test/AzureExtension/AzureExtension.Test.csproj

```diff
@@ -5,8 +5,8 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <RootNamespace>AzureExtension.Test</RootNamespace>
-    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
-    <PublishProfile>Properties\PublishProfiles\win10-$(Platform).pubxml</PublishProfile>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <PublishProfile>Properties\PublishProfiles\win-$(Platform).pubxml</PublishProfile>

```

Comment:
Updating `PublishProfile` means you also want to rename these files and the runtime identifier in them: https://github.com/microsoft/DevHomeAzureExtension/tree/user/dhoehna/MovingToDotNETEight/src/AzureExtensionServer/Properties/PublishProfiles

Reply:
Nvm, noticed this is the test project, which doesn't look to be a properties folder?

Reply:
Yeah.  I would thinf changing <RuntimeIdentifiers> and <PublishProfile> goes hand-in hand.

Reply:
The following csproj was actually untouched, so that specific publish profile might be fine as is: https://github.com/microsoft/DevHomeAzureExtension/blob/user/dhoehna/MovingToDotNETEight/src/AzureExtensionServer/AzureExtensionServer.csproj